src/main/java/asthux/EBFF/repository/GameRecordRepository.java

```diff
@@ -0,0 +1,13 @@
+package asthux.EBFF.repository;
+
+import asthux.EBFF.domain.game.GameRecord;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface GameRecordRepository extends JpaRepository<GameRecord, Long> {
+
+  @Query("select gr from GameRecord gr inner join PlayerInfo gpi where gpi.playerName = :playerName AND gr.gameRecordId = gpi.gameRecord.gameRecordId")
```

Comment:
```suggestion
  @Query("select gr from GameRecord gr inner join PlayerInfo gpi where gpi.playerName = :playerName")
```
이렇게 하면 쿼리 어떻게 나가나요? 똑같을거같은데 확인 한 번만 해주세요.

Reply:
a0185924ed3cb7c391a26fd4dde3795577ee8c35 쿼리 변경 완료